src/payment/offsite-payment-mappers/payload-mapper.js

```diff
@@ -87,6 +88,12 @@ export default class PayloadMapper {
             this.storeMapper.mapToStore(data)
         );
 
+        if (payment && payment.instrumentId) {
```

Comment:
Since we already have `formattedPayload` as part of this endpoint, could we just use it for this?

Something like this in the SDK:

```ts
formattedPayload: {
	bigpay_token: payment.instrumentId,
	vault_payment_instrument: payment.shouldSaveInstrument,
}
```

Reply:
@icatalina  I made the changes you requested and also modified the SDK, could you please take a look?
https://github.com/bigcommerce/checkout-sdk-js/pull/742

Reply:
Thanks, could you amend the PR to reflect the actual change being made and the reasons for it? It still reads as _instrument vaulting for barclays_ but we are just adding customerId (not sure why)

Reply:
Done @icatalina , I've also quashed the commits.